.github/workflows/tests-sdk.yml

```
@@ -0,0 +1,33 @@
+    branches: main
+name: SDK Tests
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v5
+        with:
+          go-version: 1.23
+      - uses: actions/checkout@v4
+      - name: Extract Tag or Commit
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:7:3: SDK_VERSION appears unused. Verify use (or export if used externally)

(shellcheck)